src/component/organism/box-slider-menu/box-slider-menu.tsx

```diff
@@ -1,16 +1,16 @@
 import { FC, useReducer } from "react";
 import TextCarousel from "../text-carousel/text-carousel";
 import MenuWrapper from "../../molecules/menu-wrapper/menu-wrapper";
-import { CategoriesModel } from "../../../api/api-get-categories";
 import { useMediaQuery } from "@mui/material";
 import { breakpointViewSize } from "../../../state/state";
 import { reducer, initState, REDUCER_ACTION_TYPE } from "../../../reducer/navbar-reducer";
+import { ICategories } from "../../../api/api-categories/api-categories";
 
-interface PropsModel {
-  categories: CategoriesModel[];
+interface ISliderMenu {
```

Comment:
When you declaring type for props then something like `IMyTypeProps` would be enough :) I mean it is ok to have `props` word in a type name